### ISSUE TITLE
Won't build?

### ISSUE THREAD

**Issue by plettb** [syncfusion/maui-demos#19]
Using VS2022 17.2.0 Preview 2.0, if I just download the source (either Git or zip) and try to build/run it I get:

`C:\Research\maui-demos-master\SampleBrowser.Maui.Core\Controls\WrapLayout\WrapLayout.cs(64,49,64,58): error CS0246: The type or namespace name 'Rectangle' could not be found (are you missing a using directive or an assembly reference?)`

If I fix "WrapLayout.cs" (fix the override to use "Rect" rather than "Rectangle") 

`C:\Research\maui-demos-master\SampleBrowser.Maui\Samples\SfCartesianChart\Samples\Bar\BarChart.xaml.cs(95,9,95,19): error CS0246: The type or namespace name 'RectangleF' could not be found (are you missing a using directive or an assembly reference?)`

(Actually there are 7 errors due to referencing "RectangleF" - that's just the first one.)

Help?

**Comment by TiMeJuMp** [comment]
same issues. This came with the Update of .Net Maui to Preview 14. 
Since then I can't use anymore the Scheduler

**Comment by PaulAnderson02** [comment]
Hi All,

Greetings from Syncfusion.

Now the .NET MAUI preview 14 compatible support for Syncfusion MAUI SampleBrowser has been provided and also has our latest Volume 1 2022 release updates has been added. Please checkout the latest files.

**Comment by plettb** [comment]
> Now the .NET MAUI preview 14 compatible support for Syncfusion MAUI SampleBrowser has been provided and also has our latest Volume 1 2022 release updates has been added. Please checkout the latest files.

Nope, it still won't build for me.  This is an essentially new/clean install.  I'm running Version 17.2.0 Preview 3.0.  Problems:
1. `SampleBrowser.Maui.Core\Controls\TabBar\CustomGrid.cs(55,29,55,43): error CS0104: 'TouchEventArgs' is an ambiguous reference between 'Microsoft.Maui.Controls.TouchEventArgs' and 'Syncfusion.Maui.Core.Internals.TouchEventArgs'` - fairly easily fixed, but really - should I need to fix this?
2. `SampleBrowser.Maui\SampleBrowser.Maui.csproj : error NU1605: Detected package downgrade: Microsoft.Graphics.Win2D from 1.0.1 to 1.0.0.30. Reference the package directly from the project to select a different version.`  and `SampleBrowser.Maui\SampleBrowser.Maui.csproj : error NU1605: Detected package downgrade: Microsoft.WindowsAppSDK from 1.0.1 to 1.0.0. Reference the package directly from the project to select a different version.` - oddly not so easily fixed.  I had to edit the csproj directly in order to fix it.
3. `SampleBrowser.Maui\App.xaml.cs(10,22,10,32): error CS0234: The type or namespace name 'Essentials' does not exist in the namespace 'Microsoft.Maui' (are you missing an assembly reference?)` - I haven't gotten this fixed yet.  I tried a few "Microsoft.Maui.Essentials" NuGets, but none seemed to work.

**Comment by AndreasReitberger** [comment]
I opened a ticket at Syncfusion for this. The developer said RC1 has huge changes and they will come back to me on April 19 with further information.

**Comment by plettb** [comment]
> The developer said RC1 has huge changes

Yeah ok - that's a valid reason.  I noticed a surprising number of changes myself in RC1.

**Comment by AndreasReitberger** [comment]
Update should come on 26.4 according to Syncfusion 👍🏻
